<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MoteControl</class>
 <widget class="QWidget" name="MoteControl">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>682</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_Mote">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>251</width>
     <height>681</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Mote</string>
   </property>
   <widget class="QPushButton" name="pushButton_Reset">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>650</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>故障复位</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_Pump1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>111</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>1#油泵</string>
    </property>
    <widget class="QLabel" name="label_Pump1AutoCtrl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>自动控制</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Pump1Run">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>电机运行</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Pump1Fault">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>电机故障</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Pump1Mesh">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>主路合闸</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Pump1StartFailed">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>启动失败</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_CtrlPower1Ok">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>电源正常</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_OilBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>231</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>油箱</string>
    </property>
    <widget class="QLabel" name="label_OilTemp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>油温：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_OilTempVal">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>54</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_OilLevel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>油位状态</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_OilTempState">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>油温状态</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_OilHeaterOn">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>加热器投入</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_OilFilterFault">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>滤油器堵塞</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_OilTempTransmitterFault">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>油温变送器故障</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_OilTempUnit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>C</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_SysPress">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>231</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>系统压力</string>
    </property>
    <widget class="QLabel" name="label_SysPressVal">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>54</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_SysPress">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>当前压力：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_PressAlarm">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>压力状态</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_LeftCavityPipeBreak">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>左腔软管破裂</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_RightCavityPipeBreak">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>右腔软管破裂</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_CavityPipeBreak">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>80</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>有杆腔软管破裂</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_CavityPressOver">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>100</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>有杆腔超压</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_PressConverterFault">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>压力变送器故障</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_NoPoleCavityPressHigh">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>无杆腔压力过高</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_ElectromagneticPowerOk">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>电磁阀电源正常</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_SysPressUnit">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>MPa</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_Gate">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>231</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>闸门</string>
    </property>
    <widget class="QPushButton" name="pushButton_GateOpen">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>开启</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_GateConfig">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>60</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>设置</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_CurrGateOpen">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>当前开度：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_GateOpenCfgVal">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_CurrGateOpenVal">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_GateOpenCfg">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>预设开度：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_GateClose">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>90</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>关闭</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_GateStop">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>90</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>停止</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>211</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_GateOpenCfgRead">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>当前预设开度：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_GateOpenCfgReadVal">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_GateDown">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>下滑：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_GateDownVal">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>140</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_GateDeviation">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>偏差：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_GateDeviationVal">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>220</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_GateLeftTravel">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>180</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>左缸行程：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_GateRightTravel">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>200</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>右缸行程：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_GateLeftTravelVal">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>180</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_GateRightTravelVal">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>200</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_GateAutoRemoteCtrl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>自动远程</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_GateOpenClose">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>120</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>全开全关</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_GateOnOff">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>120</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>开启关闭</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_GateDownAlarm">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>140</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>下滑报警</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_GateOpenSlowAlarm">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>开启速度过低</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_GateOpenSlowStop">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>开启速度过低停机</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_GateTimeoutStop">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>启闭超时停机</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_GateDeviationOver">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>220</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>同步偏差过大</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_GateTravel">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>160</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>油缸行程：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_GateTravelVal">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>160</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_GateOpenValUnit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>m</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_GateOpenCfgValUnit">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>m</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_GateTravelValUnit">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>160</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>m</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_GateLeftTravelValUnit">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>180</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>m</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_GateRightTravelValUnit">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>200</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>m</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_GateDownValUnit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>140</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>mm</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_GateDeviationValUnit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>220</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>mm</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_GateOpenCfgValUnit_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>mm</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_Pump2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>270</y>
      <width>111</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>2#油泵</string>
    </property>
    <widget class="QLabel" name="label_Pump2AutoCtrl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>自动控制</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_CtrlPower2Ok">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>电源正常</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Pump2Fault">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>电机故障</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Pump2Run">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>电机运行</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Pump2Mesh">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>主路合闸</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Pump2StartFailed">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>启动失败</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_Ip">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>640</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>IP：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_IpAddr">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>640</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_ConnectState">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>660</y>
      <width>54</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>连接状态：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ConnectStateVal">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>660</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
